floater/utils.py

```diff
@@ -251,13 +251,13 @@ def floats_to_netcdf(input_dir, output_fname,
 
     match_pattern = float_file_prefix + '.*.csv'
     float_files = glob(os.path.join(input_dir, match_pattern))
-    header_type = type(dd.read_csv(float_files[0]).columns[0])
+    float_header = dd.read_csv(float_files[0]).columns
```

Comment:
I recommend you replace this with a pandas call which only reads two rows, i.e.
```python
pd.read_csv(float_files[0], nrows=2)
```
https://pandas.pydata.org/pandas-docs/stable/generated/pandas.read_csv.html